project-chip/connectedhomeip
# Add Telink dependencies to vscode Docker image

#### Problem
https://github.com/project-chip/connectedhomeip/issues/8843

#### Change overview
Add Telink dependencies to Dockerfile of vscode

#### Testing
Manually:
1. Build docker image 
2. Run docker image
3. Check if following command able to build examples for all platforms:
```$ ./scripts/build/build_examples.py --platform all build```
